server/app/controllers/dev/seeding/DevDatabaseSeedTask.java

```diff
@@ -423,6 +427,42 @@ private void inSerializableTransaction(Runnable fn, int tryCount) {
     }
   }
 
+  public void seedApplications(String programSlug, int count) {
```

Comment:
```suggestion

  /** Seed the specified program with {@code count} number of empty applications. */
  public void seedApplications(String programSlug, int count) {
```

(blocking) Add javadoc please.